unittests/asmMocks.c

```diff
@@ -0,0 +1,55 @@
+void isr0() { }
```

Comment:
Yeah I had something like that in mind when we talked about making the code more testable.